services/skus/service.go

```diff
@@ -1976,14 +1977,72 @@ func (s *Service) createStripeSessID(ctx context.Context, req *model.CreateOrder
 	return sess.SessionID, nil
 }
 
-// TODO: Refactor the Radom-related logic.
 func (s *Service) createRadomSessID(ctx context.Context, req *model.CreateOrderRequestNew, order *model.Order) (string, error) {
-	sess, err := order.CreateRadomCheckoutSession(ctx, s.radomClient, s.radomSellerAddress)
+	oid := order.ID.String()
+
+	surl, err := req.RadomMetadata.SuccessURL(oid)
 	if err != nil {
 		return "", err
 	}
 
-	return sess.SessionID, nil
+	curl, err := req.RadomMetadata.CancelURL(oid)
+	if err != nil {
+		return "", err
+	}
+
+	items, err := orderItemsToLineItems(order.Items)
+	if err != nil {
+		return "", err
+	}
+
+	reqx := radom.CheckoutSessionRequest{
+		LineItems:  items,
+		Gateway:    s.radomGateway,
+		SuccessURL: surl,
+		CancelURL:  curl,
+		Metadata: []radom.Metadata{
+			{
+				Key:   "brave_order_id",
+				Value: oid,
+			},
+		},
+		ExpiresAt: time.Now().Add(24 * time.Hour).Unix(),
```

Comment:
The 24 hrs is added keep the checkout expiry consistent with the other payment providers. Will add UTC. I don't think the extra indirection is necessary atm moment as it just gets passed in and we don't need to assert on it in the tests.

Reply:
Is the call to UTC necessary given the `.Unix()` returns the seconds elapsed since Jan 1970 UTC. 
> The result does not depend on the location associated with t.